levelpacker/levelpacker.py

```diff
@@ -267,26 +272,37 @@ def process_level(path: str):
     tilelayer = 0
     builder = flatbuffers.Builder(1)
 
+    processedLayers = 0
+
     for ts in map_node.getElementsByTagName('tileset'):
         handle_tileset(ts, objs, builder)
     if map_node.getElementsByTagName('layer'):
         tilelayer = handle_tile_layer(map_node.getElementsByTagName('layer')[0], builder)
+        processedLayers += 1
 
     for g in map_node.getElementsByTagName('objectgroup'):
         group_name = g.getAttribute('name')
         if group_name == 'objects':
             handle_objects_layer(g, objs, builder)
+            processedLayers += 1
         elif group_name == 'decoration':
             handle_decoration_layer(g, objs, builder)
+            processedLayers += 1
         elif group_name == 'collisionMasks':
             handle_collision_layer(g, objs, builder)
+            processedLayers += 1
         elif group_name == 'hidingSpots':
             handle_hidingspots_layer(g, objs, builder)
+            processedLayers += 1
         elif group_name == 'meta':
             handle_meta_layer(g, objs, builder)
+            processedLayers += 1
         else:
             print("WARNING: Unknown group {}".format(group_name))
     
+    if processedLayers < 6:
+        print("WARNING: a layer is missing")
+
```

Comment:
This might actually be a mistake on my side - I wanted to make it so that a missing layer (tile layer e.g.) would be acceptable, but it seems I have failed

Reply:
I think that when you were writing this it actually worked as you expected. However due to later changes that I've introduced the game started to crash when it couldn't find certain layers.